data/mods/Generic_Guns/gg_ammo_hand_override.json

```diff
@@ -5,7 +5,7 @@
     "type": "AMMO",
     "name": "reloaded light rifle ammo",
     "name_plural": "reloaded light rifle ammo",
-    "description": "Hand-loaded ammunition for lighter rifles, designed for small game and humans.",
+    "description": "Hand-loaded .223 ammunition for lighter rifles, designed for small game and humans.",
```

Comment:
I thought these were suppose to be generic and not be number specific. This is part of the Generic guns mod.

Reply:
Whoops. Reverted.